### DISCUSSION TITLE
How are shipping labels handled?

### DISCUSSION THREAD

**Discussion by christopherarter** [duncanmcclean/simple-commerce#1070] [Help] [answered]
Pre-purchase question!

I was wondering if simple-commerce handled purchasing & printing shipping labels in the admin? (similar to WooCommerce & Shopify)?

Or, generally how users are handling that part of the process? Are they subscribing to a separate service and printing from there?

Thanks!

**Comment by duncanmcclean** [comment, marked as answer]
That's a great question! 

I *think* most folks might be using a separate service and copying the address details over there. You could fairly easily build your own action in the CP to generate & download them or hook into the [`PaymentStatusUpdated` event](https://simple-commerce.duncanmcclean.com/events#content-paymentstatusupdated), check for when an order is paid, and do some kind of API request to a third-party.
